roles/alexandria-v2/templates/application.yml.j2

```diff
@@ -1,17 +1,21 @@
 defaults: &defaults
   download_root: 'tmp/download_root'
-  pegasus_sru_endpoint: 'http://pegasus-test.library.ucsb.edu:5661'
-  marmotta_host: localhost:8180
   internal_local_vocab_root: <%= ActiveFedora.fedora.host %>
   minter_state: 'tmp/minter-state'
+  pegasus_sru_endpoint: 'http://pegasus-test.library.ucsb.edu:5661'
+
 development:
   <<: *defaults
+  marmotta_host: {{ server_name }}:8080
+
 test:
   <<: *defaults
+  marmotta_host: localhost:8180
+
 production:
   <<: *defaults
-  marmotta_host: {{ marmotta_host }}
-  pegasus_sru_endpoint: 'http://pegasus.library.ucsb.edu:5661'
+  download_root: {{ etd_download_root }}
   internal_local_vocab_root: 'http://alexandria.ucsb.edu:8080/fedora/rest'
+  marmotta_host: {{ server_name }}:8080
```

Comment:
@dunn If marmotta is hard-configured to port 8080, we should remove the marmotta-jetty task. Any reason to keep it?